Add `repository.url` and `repository.provider` labels (#97)
autometrics-dev/autometrics-py: README.md

>
> autometrics-py will track support for build_info using the OpenTelemetry tracker via [this issue](https://github.com/autometrics-dev/autometrics-py/issues/38)

The library uses a separate metric (`build_info`) to track the version and git metadata of your code - repository url, provider name, commit and branch.

> It then writes queries that group metrics by the set labels so you can spot correlations between code changes and potential issues.

Reviewer: reword: "It then writes queries that group metrics by these labels, so you can spot correlations between code changes and potential issues."

Reviewer: holdup, i forgot i can suggest a change directly in github, gonna do that

> It then writes queries that group metrics by the set labels so you can spot correlations between code changes and potential issues.

Reviewer: ```suggestion
It then writes queries that group metrics by these metadata, so you can spot correlations between code changes and potential issues.
```

Configure these labels by setting the following environment variables:

| Label                 | Run-Time Environment Variables        | Default value |
| --------------------- | ------------------------------------- | ------------- |
| `version`             | `AUTOMETRICS_VERSION`                 | `""`          |
| `commit`              | `AUTOMETRICS_COMMIT` or `COMMIT_SHA`  | `""`          |
| `branch`              | `AUTOMETRICS_BRANCH` or `BRANCH_NAME` | `""`          |
| `repository_url`      | `AUTOMETRICS_REPOSITORY_URL`          | `""`\*        |
| `repository_provider` | `AUTOMETRICS_REPOSITORY_PROVIDER`     | `""`\*        |

> \* Autometrics will attempt to automagically infer these values from git config inside working directory. To disable this behavior, explicitly set the corresponding setting or environment variable to `""`.

Reviewer: ```suggestion
\* Autometrics will attempt to automagically infer these values from the git config inside your working directory. To disable this behavior, explicitly set the corresponding setting or environment variable to `""`.
```
